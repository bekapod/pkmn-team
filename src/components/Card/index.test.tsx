import React from "react";
import renderer from "react-test-renderer";
import { CardWrapper, CardHeader, CardContent } from ".";

describe("Card components", () => {
  it("renders without crashing", () => {
    const tree = renderer.create(
      <CardWrapper>
        <CardHeader types={["ELECTRIC", "PSYCHIC"]}>Heading</CardHeader>
        <CardContent>Content</CardContent>
      </CardWrapper>
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
