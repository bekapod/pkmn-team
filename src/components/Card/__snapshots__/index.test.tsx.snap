// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card components renders without crashing 1`] = `
<article
  className="Card__CardWrapper-sc-3o7tcb-0 bTjMad"
>
  <header
    className="Card__CardHeader-sc-3o7tcb-1 dklHtH"
  >
    Heading
  </header>
  <div
    className="Card__CardContent-sc-3o7tcb-2 fDtTnd"
  >
    Content
  </div>
</article>
`;
